controllers/drplacementcontrol.go

```diff
@@ -30,6 +30,9 @@ const (
 	// Annotations for MW and PlacementRule
 	DRPCNameAnnotation      = "drplacementcontrol.ramendr.openshift.io/drpc-name"
 	DRPCNamespaceAnnotation = "drplacementcontrol.ramendr.openshift.io/drpc-namespace"
+
+	// Annotation for the last cluster on which the application was running
+	LastAppDeploymentCluster = "drplacementcontrol.ramendr.openshift.io/last-app-deployment-cluster"
```

Comment:
The name is not consistent with other annotations above but I guess the reason is
DRPCName*Annotation is annotation for ManifestWork and PlacementRule (and Placement?) and this annotation is for the drpc resource. Maybe having a separate `const ()` with a comment for drpc annotations will make this more clear.

Regarding the new annotation name - how it is related to preferredCluster and failoverCluster?

```
  spec:
    failoverCluster: dr2
    preferredCluster: dr1
    ...
```

Isn't it the same as preferredCluster during failover, and failoverCluster during relocate?

Regardless, can we use something more consistent with these names, like last-cluster? or last-primary-cluster?

Reply:
The name was proposed by the UX team, and I find it quite logical. Essentially, `last-app-deployment-cluster` refers to the cluster where the app was last deployed on.